packages/cardpay-sdk/sdk/utils/general-utils.ts

```diff
@@ -14,20 +14,24 @@ export async function networkName(web3: Web3): Promise<string> {
   return name;
 }
 
-export function waitUntilTransactionMined(web3: Web3, txnHash: string): Promise<TransactionReceipt> {
+export function waitUntilTransactionMined(web3: Web3, txnHash: string, duration = 30000): Promise<TransactionReceipt> {
```

Comment:
this is more of a style thing, but lots of times its nice to show the math. so instead of `30000` you can say `1000 * 60 * 5`, then it's a little more clear we mean 5 minutes.